packages/components/pages/f-checkout/src/components/_tests/Accordion.test.js

```diff
@@ -0,0 +1,103 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Accordion from '../Accordion.vue';
+import { i18n, createStore } from './helpers/setup';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('Accordion', () => {
+    const propsData = { title: 'Accordion Header', id: 'note' };
+
+    it('should be defined', () => {
+        // Arrange & Act
+        const wrapper = shallowMount(Accordion, {
+            i18n,
+            store: createStore(),
+            localVue,
+            propsData
+        });
+
+        // Assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    describe('methods ::', () => {
+        describe('toggle ::', () => {
+            let wrapper;
+
+            it('should change `isExpanded` to true when it is initally set to false', () => {
+                // Arrange
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: false
+                        };
+                    }
+                });
+
+                // Act
+                wrapper.vm.toggle();
+
+                // Assert
+                expect(wrapper.vm.$data.isExpanded).toBe(true);
+            });
+
+            it('should change `isExpanded` to false when it is initally set to true', () => {
+                // Arrange
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: true
+                        };
+                    }
+                });
+
+                // Act
+                wrapper.vm.toggle();
+
+                // Assert
+                expect(wrapper.vm.$data.isExpanded).toBe(false);
+            });
+        });
```

Comment:
You can combine these two tests with `it.each`
```suggestion
            it.each([
                [true, false],
                [false, true]
            ])('should change `isExpanded` to %s when it is initially set to %s', (expected, initial) => {
                // Arrange
                wrapper = shallowMount(Accordion, {
                    i18n,
                    store: createStore(),
                    localVue,
                    propsData,
                    data () {
                        return {
                            isExpanded: initial
                        };
                    }
                });

                // Act
                wrapper.vm.toggle();

                // Assert
                expect(wrapper.vm.$data.isExpanded).toBe(expected);
            });
        });
```